style.css

```diff
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+.content {
+  background: #fff;
+  height: 80px;
+  width: 100%;
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-branding {
+  font-size: 20px;
+  font-family: "Poppins", sans-serif;
+  color: #6070ff;
+  font-weight: 700;
+  line-height: 40px;
+  padding: 0 30px;
+  cursor: pointer;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 50px;
+}
+
+.nav-item {
+  display: none;
+}
+
+.content2 {
+  display: block;
+  cursor: pointer;
+  margin-right: 20px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px 0;
+  transition: all 0.3s ease;
+  background-color: #6070ff;
+}
+
+.section1 {
+  background-image: url("assets/images/background-image.svg");
+  background-position: top;
```

Comment:
- [ ] OPTIONAL: Please make sure your background image is displayed. Actually, your image doesn't exist. It may be caused by the image files you deleted. Kindly add it to stick to Figma template.